environments/swiss/public.yml

```diff
@@ -14,7 +14,7 @@ elastcsearch_backup_days: 2
 
 redis_maxmemory: 4gb
 
-ssh_allow_password: True
+ssh_allow_password: False
```

Comment:
what's the reason for this change? I'm assuming this was enabled for someone on the swiss team to be able to access the server. just making sure they know about this.

Reply:
Yeah I asked Gonçalo about it and he confirmed that I could change it to use our recommended setting here